src/app/modules/question-bank/components/download-questions/download-questions.component.html

```diff
@@ -0,0 +1,85 @@
+<div class="container">
+  <div class="card">
+    <div class="d-flex justify-content-center">
+      <h2 class="p-2">Download Questions Form</h2>
+    </div>
+    <form class="p-5 card-body" [formGroup]="downloadForm">
+      <div class="form-group mt-3">
+        <label for="tech_skills"> Technical Skills: </label>
+        <ng-multiselect-dropdown
+          [placeholder]="'Select Technical Skills'"
+          [settings]="dropdownSettings"
+          [data]="skillsList"
+          formControlName="techSkills"
+        >
+        </ng-multiselect-dropdown>
+        <small
+          class="text-danger"
+          *ngIf="
+            downloadForm.get('techSkills')?.invalid &&
+            (downloadForm.get('techSkills')?.dirty || downloadForm.get('techSkills')?.touched)
+          "
+        >
+          Please provide technical skills.
+        </small>
+      </div>
+      <div class="form-group mt-3">
+        <label for="tech_experience"> Technical Experience: </label>
+        <div *ngFor="let exp of yearsOfExperience; let i = index">
+          <input
+            type="radio"
+            id="tech_experience"
+            formControlName="techExperience"
+            id="radio{{ i }}"
+            name="techExperience"
+            [value]="exp.id"
+          />
+          <label class="custom-control-label m-1" for="radio{{ i }}">
+            {{ exp.years || '?' }}
+          </label>
+        </div>
+        <small
+          class="text-danger"
+          *ngIf="
+            downloadForm.get('techExperience')?.invalid &&
+            (downloadForm.get('techExperience')?.dirty || downloadForm.get('techExperience')?.touched)
+          "
+        >
+          Please provide years of technical experience.
+        </small>
+      </div>
+      <div class="form-group mt-3">
+        <label for="questionsCount"> No. of Questions </label>
+        <input
+          id="questionsCount"
+          class="form-control"
+          formControlName="questionsCount"
+          placeholder="Questions Count"
+        />
+        <small
+          *ngIf="
+            downloadForm.get('questionsCount')?.invalid &&
+            (downloadForm.get('questionsCount')?.dirty || downloadForm.get('questionsCount')?.touched)
+          "
+        >
+          Please provide Questions to be downloaded.
+        </small>
```

Comment:
Can we also display this error msg in red like technical skills
<img width="1236" alt="image" src="https://user-images.githubusercontent.com/71638717/192253792-94b43ca9-cc7d-43d7-97be-2663f7101f65.png">